lib/ui/views/farm_deposit/bloc/provider.dart

```diff
@@ -166,41 +154,39 @@ class FarmDepositFormNotifier
     return true;
   }
 
-  Future<void> deposit(BuildContext context, WidgetRef ref) async {
+  Future<void> deposit(AppLocalizations localizations) async {
```

Comment:
Why don't we use a provider for AppLocalizations ?

Reply:
We could do this.

Reply:
ok. if we do this, i prefer in an issue later. very low priority